ui/app/models/kv/metadata.js

```diff
@@ -71,17 +71,14 @@ export default class KvSecretMetadataModel extends Model {
   @lazyCapabilities(apiPath`${'backend'}/data/${'path'}`, 'backend', 'path') dataPath;
   @lazyCapabilities(apiPath`${'backend'}/metadata/${'path'}`, 'backend', 'path') metadataPath;
 
-  get canListMetadata() {
-    return this.metadataPath.get('canList');
-  }
   get canDeleteMetadata() {
     return this.metadataPath.get('canDelete');
   }
   get canReadMetadata() {
     return this.metadataPath.get('canRead');
   }
-  get canReadData() {
-    return this.dataPath.get('canRead');
+  get canUpdateMetadata() {
+    return this.metadataPath.get('canUpdate');
   }
   get canEditData() {
     return this.dataPath.get('canUpdate');
```

Comment:
Should we update this to `this.metadataPath.get('canUpdate')`?

Reply:
This model is calling both the data and metadata, so I need the canUpdate for metadata above it and the one below it for data.

Reply:
That makes sense thanks for clarifying!